addons/auth_totp/models/res_users.py

```diff
@@ -79,6 +79,61 @@ def _totp_try_setting(self, secret, code):
         _logger.info("2FA enable: SUCCESS for %s %r", self, self.login)
         return True
 
+    def get_totp_invite_url(self, user):
+        return '/web#action=auth_totp.action_activate_two_factor_authentication'
+
+    def totp_invite(self):
+        mails_to_send = self.env['mail.mail']
+        invite_template = self.env.ref('auth_totp.mail_template_totp_invite', raise_if_not_found=False)
+        for user in self.sudo().filtered(lambda user: not user.totp_secret):
+            ctx = {
+                'author_name': self.env.user.partner_id.name,
```

Comment:
``user`` is already available in rendering context, so you could use ``user.name`` instead of ``author_name``